docs/FAQ.md

```diff
@@ -0,0 +1,45 @@
+# **<center>FAQ'S</center>**
+
+**Q1.** Who all can use this website?
+
+**Ans:** a)**Admin:** For managing the website.                      
+         b)**Transplant Coordinator:** A doctor from a registered hospital who performs a kidney transplant.  
+
+**Q2.** How to setup the project on your local machine?
+
+**Ans:**  The [build documentation](https://github.com/siv2r/kidney-exchange/blob/master/docs/build.md)  provided in GitHub explains this in detail.
+
+**Q3.** What credentials are required to log in as a transplant coordinator?
+
+
+**Ans:** Username:**test1** 
+  
+   Password: **test1**
+
+
+**Q4.** What credentials are required to log in as admin?
+
+**Ans:** Username:**admin1**
+ 
+ Password:**kep1**
+     
+
+
+**Q5.** Why do the credentials provided in Q3 and Q4 are not working?
+
+**Ans:** During login, the script checks the users’ table on the database to see if the provided login credentials are correct. Your database might be empty, so this will fail. Therefore, you must import filled_kidney_exchange_data.sql from the [manual setup](https://www.notion.so/Project-setup-Public-1a647ed8515c485f99f38e717acfa61b) document for the credential provided in Q3, Q4 to work.
```

Comment:
bound the filled_kidney_exchange_data.sql with backticks so, it looks like this `filled_kidney_exchange_data.sql`

Reply:
okay i will do this change till morning

Reply:
Sure no worries :)